src/core/email/templates/progress-report-status-changed.template.tsx

```diff
@@ -0,0 +1,91 @@
+import {
+  Button,
+  Column,
+  HideInText,
+  Section,
+  Text,
+} from '@seedcompany/nestjs-email/templates';
+import { startCase } from 'lodash';
+import { DateTime } from 'luxon';
+import { EmailReportStatusNotification } from '../../../components/progress-report/workflow/progress-report-workflow.service';
+import { fullName } from '../../../components/user/fullName';
+import { EmailTemplate, Heading } from './base';
+import { useFrontendUrl } from './frontend-url';
+
+export function ProgressReportStatusChanged({
+  changedBy,
+  project,
+  languageEngagement,
+  report,
+  reportQtrLabel,
+  newStatusVal,
+  previousStatusVal,
+}: EmailReportStatusNotification) {
+  const projectUrl = project?.id
+    ? useFrontendUrl(`/projects/${project.id}`)
+    : useFrontendUrl(`/projects/no-project-found`);
+  const projectName = project?.name.value;
+
+  const reportUrl = report.id
+    ? useFrontendUrl(`/progress-reports/${report.id}`)
+    : useFrontendUrl(`/projects/no-project-reports-found`);
+  const reportLabel = reportQtrLabel
+    ? `Quarterly Report - ${reportQtrLabel}`
+    : 'Quarterly Report';
+
+  const changerUrl = useFrontendUrl(`/users/${changedBy.id}`);
+  const changerName = fullName(changedBy);
+
+  const oldStatus = startCase(previousStatusVal) || undefined;
+  const newStatus = startCase(newStatusVal) || undefined;
+
+  const changedAt = DateTime.now()
+    .setZone(changedBy.timezone.value ?? 'America/Chicago')
+    .toLocaleString(DateTime.DATETIME_FULL);
+
+  return (
+    <EmailTemplate
+      title={
+        reportLabel && oldStatus && newStatus
+          ? `${reportLabel} changed from ${oldStatus} to ${newStatus}`
+          : `${reportLabel} Status Change`
+      }
+    >
+      <Heading>
+        {reportLabel && newStatus ? (
+          <>
+            {reportLabel} is now <em>{newStatus}</em>
+          </>
+        ) : (
+          `${reportLabel} has a new status`
+        )}
+      </Heading>
+
+      <Section>
+        <Column>
+          <Text paddingBottom={16}>
+            {changerName ? <a href={changerUrl}>{changerName}</a> : 'Someone'}{' '}
+            has changed <a href={reportUrl}>{reportLabel}</a>{' '}
+            {newStatus ? (
+              <>
+                {oldStatus ? (
+                  <>
+                    from <em>{oldStatus}</em>{' '}
+                  </>
+                ) : null}
+                to <em>{newStatus}</em>{' '}
+              </>
+            ) : null}
+            for <a href={projectUrl}>{projectName}</a> - {languageEngagement} on{' '}
+            {changedAt}
+          </Text>
+          <HideInText>
```

Comment:
I don't think there's a need to hide this button in text. It will be converted by a convention.
```suggestion
```